web/src/compositions/usePipeline.ts

```diff
@@ -81,7 +81,15 @@ export default (pipeline: Ref<Pipeline | undefined>) => {
     return convertEmojis(pipeline.value.message);
   });
 
-  const title = computed(() => message.value.split('\n')[0]);
+  const title = computed(() => {
+    if (!pipeline.value) {
+      return '';
+    }
+    if (pipeline.value.title !== '') {
+      return convertEmojis(pipeline.value.title);
+    }
```

Comment:
This will need a dedicated UI element as the `title` is the tooltip showing the not truncated message.